src/bioversions/sources/rhea.py

```diff
@@ -13,7 +13,7 @@
 
 requests_ftp.monkeypatch_session()
```

Comment:
do you want to take this monkeypatch out? I guess we don't need it here anymore

Reply:
Done, wasn't sure if it served any other purpose.